python/hail/__init__.py

```diff
@@ -1,13 +1,13 @@
-from hail.representation import *
-from hail.context import HailContext
-from hail.dataset import VariantDataset
-from hail.typ import Type, TInt32, TInt64, TFloat32, TFloat64, TSet, TString, TBoolean, TArray, TDict, TLocus, TVariant, TAltAllele, TCall, TInterval, TStruct
-from hail.keytable import KeyTable
-from hail.kinshipMatrix import KinshipMatrix
-from hail.ldMatrix import LDMatrix
-from hail.utils import hadoop_read, hadoop_write, hadoop_copy
-
 import sys
+
+from hail.api1.context import HailContext
+from hail.api1.dataset import VariantDataset
+from hail.api1.keytable import KeyTable
```

Comment:
in this case, yes. I don't want to do that everywhere, though, since if `hail.api1.__init__.py` imports a bunch of stuff, then all of that will be tab-completable if you do 
```python
import hail
hail.foo # something imported in hail.api.__init__
```